contracts/PoolManager.sol

```diff
@@ -380,14 +366,14 @@ contract PoolManager is IPoolManager, Fees, NoDelegateCall, ERC1155, IERC1155Rec
     }
 
     function setProtocolFees(PoolKey memory key) external {
-        (uint8 newProtocolSwapFee, uint8 newProtocolWithdrawFee) = _fetchProtocolFees(key);
+        (uint16 newProtocolSwapFee, uint16 newProtocolWithdrawFee) = _fetchProtocolFees(key);
         PoolId id = key.toId();
         pools[id].setProtocolFees(newProtocolSwapFee, newProtocolWithdrawFee);
         emit ProtocolFeeUpdated(id, newProtocolSwapFee, newProtocolWithdrawFee);
     }
 
     function setHookFees(PoolKey memory key) external {
-        (uint8 newHookSwapFee, uint8 newHookWithdrawFee) = _fetchHookFees(key);
+        (uint16 newHookSwapFee, uint16 newHookWithdrawFee) = _fetchHookFees(key);
```

Comment:
same here.. can just return uint24 `newHookFee`

Reply:
and then you can also change the interface for setHookFees, setProtocolFees, and also initialize